erpnext/accounts/doctype/coupon_code/test_coupon_code.py

```diff
@@ -57,7 +57,7 @@ def test_create_test_data():
 		})
 		item_price.insert()
 	# create test item pricing rule
-	if not frappe.db.exists("Pricing Rule","_Test Pricing Rule for _Test Item"):
+	if not frappe.db.exists("Pricing Rule",{"title": "_Test Pricing Rule for _Test Item"}):
```

Comment:
Space after comma is the norm
```suggestion
	if not frappe.db.exists("Pricing Rule", {"title": "_Test Pricing Rule for _Test Item"}):
```